src/dispatcher.py

```diff
@@ -1,31 +0,0 @@
-import argparse
-
-import src.constants as const
-from src.ingestion import bulk_ingest
-from src.services import generate_recon_report, generate_ror_report
```

Comment:
Are these really services? Given the method is called `run_tasks`, maybe they should also be called "tasks"?